test/JDBC/upgrade/latest/schedule

```diff
@@ -409,3 +409,4 @@ BABEL-733
 BABEL-3938
 BABEL-NEXT-VALUE-FOR
 binary-index
+test_windows_sp_helpuser
```

Comment:
should add test which make sure that setting this GUC to false disables windows login

Reply:
Do not need to add test as discussed.